addons/io_hubs_addon/components/operators.py

```diff
@@ -500,6 +502,74 @@ def execute(self, context):
         return {'FINISHED'}
 
 
+class OpenImage(Operator):
+    bl_idname = "image.hubs_open_image"
+    bl_label = "Open Image"
+    bl_options = {'REGISTER', 'UNDO'}
+
+    filepath: StringProperty(subtype="FILE_PATH")
+    files: CollectionProperty(type=PropertyGroup)
+    filter_folder: BoolProperty(default=True, options={"HIDDEN"})
+    filter_image: BoolProperty(default=True, options={"HIDDEN"})
+    target_property: StringProperty()
+
+    relative_path: BoolProperty(
+        name="Relative Path", description="Select the file relative to the blend file", default=True)
+
+    disabled_message = "Can't open/assign images to linked data blocks. Please make it local first"
+
+    @ classmethod
+    def description(cls, context, properties):
+        description_text = "Load an external image "
+        ob = getattr(context, 'host')
+        if bpy.app.version < (3, 0, 0) and is_linked(ob):
```

Comment:
Annoyingly, the host attribute isn't getting passed to descriptions for me in Blender versions below 3.5, but it isn't really needed for versions 3.0 and up, so it'll mostly work if we utilize the if statement's short circuiting on 3.0+ (sadly, it looks like 2.93 won't work with the context pointer and so won't get the description).
`getattr` also isn't needed here, and I prefer not to use the variable name `ob` for hosts because it adds misleading preconceptions of what type the host is (I know it's a small thing, but I find it's really easy to forget that hosts can be things other than objects).
```suggestion
        if bpy.app.version < (3, 0, 0) and is_linked(context.host):
```